src/mixer/playermanager.cpp

```diff
@@ -440,6 +446,8 @@ void PlayerManager::addMicrophoneInner() {
     QString group = groupForMicrophone(index);
     Microphone* pMicrophone = new Microphone(this, group, index, m_pSoundManager,
                                              m_pEngine, m_pEffectsManager);
+    connect(pMicrophone, SIGNAL(noMicrophoneInputConfigured()),
```

Comment:
Why is this in PlayerManager? Is a Microphone considered a deck/player from its perspective?